src/Tribe/Repository.php

```diff
@@ -1226,6 +1227,7 @@ public function where_args( array $args ) {
 	public function by_args( array $args ) {
 		foreach ( $args as $key => $value ) {
 			$this->by( $key, $value );
+			$foo = 'bar';
```

Comment:
Is this meant to stay in the code, or was just for debugging?

Reply:
Ah ehm... ok... thanks.